src/ocamlorg_frontend/pages/problems.eml

```diff
@@ -48,12 +53,22 @@ Learn_layout.render
         problems is available on
         <a href="https://github.com/ocaml/ocaml.org/tree/main/data/problems">GitHub</a>.
       </p>
+      <p>
+        Every exercise has a difficulty level, ranging from beginner to advanced. 
+        This difficulty level is indicated with the symbol "☡".
+        The absence of marks represents the beginner level, one mark the intermediate level and two marks the advanced level.
+      </p>
     </div>
     <% problems |> List.iter (fun (problem : Ood.Problem.t) -> %>
     <div x-data="{statement: true}" id="<%s" problem.number %>
       >
       <div class="flex space-y-4 lg:space-y-0 lg:items-center flex-col lg:flex-row lg:justify-between">
-        <h4 class="font-bold text-body-600"><%s problem.title %></h4>
+        <span aria-hidden="true" class="danger invisible lg:visible"><%s danger problem.difficulty %></span>
+        <h4 class="font-bold text-body-600">
+          <%s problem.title %>
+          <span aria-hidden="true" class="visible lg:invisible" > <%s danger problem.difficulty %></span>
```

Comment:
```suggestion
          <span aria-hidden="true" class="ml-2 visible lg:invisible" > <%s danger problem.difficulty %></span>
```

|before|after|
|-|-|
|![Screenshot 2023-03-15 at 12-18-17 Exercises](https://user-images.githubusercontent.com/6594573/225298134-0f8fd10b-c686-4aa1-a974-e303361c3f6c.png)|![Screenshot 2023-03-15 at 12-18-25 Exercises](https://user-images.githubusercontent.com/6594573/225298144-458ad451-af78-42fb-aee2-c80e188f3b51.png) add spacing between title and symbol|